File: services/monolith/src/main/java/se/shrly/examplecg/service/monolith/Monolith.java
package se.shrly.examplecg.service.monolith;

import java.util.Map;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication(scanBasePackages = "se.shrly.externalpsp.service")
public class Monolith {

    public static void main(String[] args) {
        var app = new SpringApplication(Monolith.class);

        app.setDefaultProperties(Map.of("spring.config.additional-location", "file:./config/"));

        app.run(args);
    }
}

File: services/monolith/src/main/kotlin/se/tobiaslafleur/examplecg/service/monolith/Monolith.kt
package se.tobiaslafleur.examplecg.service.monolith

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication(scanBasePackages = ["se.tobiaslafleur.examplecg.service"])
class Monolith

fun main(args: Array<String>) {
    SpringApplication(Monolith::class.java).run(*args)
}
